README.md

```diff
@@ -1,20 +1,40 @@
 # Demo
 
-To start your Phoenix server:
+## Spec
```

Comment:
Where do you see this spec going? Will it be in the Readme of phx_gen_auth, in the moduledoc of generated code, or just in this PR?

Reply:
I originally thought it would only be part of this PR but I think you will be more capable of assessing it. If you feel like there is important information  in here, then maybe some of it should be added to `mix phx.gen.auth` docs OR added to the module docs of the related modules, especially the `user` and `user_token` ones. WDYT?

Reply:
Sounds good. I'll look for a good place to put it. :+1: